app/environments/brassbirmingham/brassbirmingham/envs/consts.py

```diff
@@ -0,0 +1,520 @@
+        LocationCard(DERBY),
+        LocationCard(LEEK),
+        LocationCard(LEEK),
+        LocationCard(STOKE_ON_TRENT),
+        LocationCard(STOKE_ON_TRENT),
+        LocationCard(STOKE_ON_TRENT),
+        LocationCard(STONE),
+        LocationCard(STONE),
+        LocationCard(UTTOXETER),
+        LocationCard(UTTOXETER),
+        LocationCard(STAFFORD),
+        LocationCard(STAFFORD),
+        LocationCard(BURTON_UPON_TRENT),
+        LocationCard(BURTON_UPON_TRENT),
+        LocationCard(CANNOCK),
+        LocationCard(CANNOCK),
+        LocationCard(TAMWORTH),
+        LocationCard(WALSALL),
+        LocationCard(COALBROOKDALE),
+        LocationCard(COALBROOKDALE),
+        LocationCard(COALBROOKDALE),
+        LocationCard(DUDLEY),
+        LocationCard(DUDLEY),
+        LocationCard(KIDDERMINSTER),
+        LocationCard(KIDDERMINSTER),
+        LocationCard(WOLVERHAMPTON),
+        LocationCard(WOLVERHAMPTON),
+        LocationCard(WORCESTER),
+        LocationCard(WORCESTER),
+        LocationCard(BIRMINGHAM),
+        LocationCard(BIRMINGHAM),
+        LocationCard(BIRMINGHAM),
+        LocationCard(COVENTRY),
+        LocationCard(COVENTRY),
+        LocationCard(COVENTRY),
+        LocationCard(NUNEATON),
+        LocationCard(REDDITCH),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.coal_mine),
+        IndustryCard(CardName.coal_mine),
+        IndustryCard(CardName.coal_mine),
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.man_goods_or_cotton),  # Manufactured goods OR Cotton
+        IndustryCard(CardName.pottery),
+        IndustryCard(CardName.pottery),
+        IndustryCard(CardName.pottery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+    ],
+}
+
+STARTING_WILD_LOCATION_CARDS: List[Card] = [
+    # idk how many wilds there are
+    LocationCard(CardName.wild_location, True),
+    LocationCard(CardName.wild_location, True),
+    LocationCard(CardName.wild_location, True),
+    LocationCard(CardName.wild_location, True),
+    LocationCard(CardName.wild_location, True),
+]
+
+STARTING_WILD_BUILDING_CARDS: List[Card] = [
```

Comment:
We don't really need these. The only restriction is that "You may not perform this action if you already have a Wild card in your Hand." So we'd have to confirm the player doesn't already have a wild card in their hand, but for that, we don't need something like this